src/kOS/Screen/TermWindow.cs

```diff
@@ -108,6 +131,39 @@ public void Awake()
             DontDestroyOnLoad(gObj);
             popupEditor = (KOSTextEditPopup)gObj.GetComponent(typeof(KOSTextEditPopup));
             popupEditor.SetUniqueId(UniqueId + 5);
+            
+            customSkin = BuildPanelSkin();
+
+            GameEvents.onHideUI.Add (OnHideUI);
+			GameEvents.onShowUI.Add (OnShowUI);
```

Comment:
Odd... these two lines seem to be showing as not indented the same in here Github. They look indented the same in my editor.